docs/adguard-browser-extension/userscripts-api.md

```diff
@@ -0,0 +1,26 @@
+---
+title: User Scripts API
+sidebar_position: 8
+---
+
+:::info
+
+This article is about AdGuard Browser Extension, which safeguards only your browser. To protect your entire device, [download the AdGuard app](https://adguard.com/download.html?auto=true)
+
+:::
+
+When adding a custom filter or a user rule, you will encounter a notification message asking you to turn on the developer mode to use these features on the extension:
+
+![Notification *border](https://cdn.adtidy.org/content/Kb/ad_blocker/browser_extension/dev-mode-ext.png)
+
+This is because, from v5.2, AdGuard Browser Extension has migrated to the User Scripts API. In other words, the extension now uses script-based rules.
```

Comment:
```suggestion
This is because, starting with version 5.2, AdGuard Browser Extension has migrated to the User Scripts API. In other words, the extension now uses script-based rules.
```

Reply:
`This is because, starting with version 5.2, AdGuard Browser Extension has migrated to the User Scripts API.`
↓
`This is because, starting with version 5.2, AdGuard Browser Extension uses the User Scripts API.`